### ISSUE TITLE
Cannot start micro_sam.annotator

### ISSUE THREAD

**Issue by phamtronglam** [computational-cell-analytics/micro-sam#212]
Hi

I tried installing microsam on 2 conda environments: One in the currently working napari-gpu environment and one in a fresh python 3.8 environment. Unfortunately, from the start, micro_sam.annotator could not start. 
Could you point me to how to fix this please?
Here is the error output:
```
(micro_sam) D:\tmp>micro_sam.annotator
D:\miniconda3_windows\envs\micro_sam\lib\site-packages\torchvision\io\image.py:13: UserWarning: Failed to load image Python extension: '[WinError 127] 指定されたプロシージャが見つかりません。'If you don't plan on using image functionality from `torchvision.io`, you can ignore this warning. Otherwise, there might be something wrong with your environment. Did you have `libjpeg` or `libpng` installed before building `torchvision` from source?
  warn(
Traceback (most recent call last):
  File "D:\miniconda3_windows\envs\micro_sam\Scripts\micro_sam.annotator-script.py", line 5, in <module>
    from micro_sam.sam_annotator.annotator import main
  File "D:\miniconda3_windows\envs\micro_sam\lib\site-packages\micro_sam\sam_annotator\__init__.py", line 5, in <module>
    from .annotator import annotator
  File "D:\miniconda3_windows\envs\micro_sam\lib\site-packages\micro_sam\sam_annotator\annotator.py", line 13, in <module>
    from .annotator_2d import annotator_2d
  File "D:\miniconda3_windows\envs\micro_sam\lib\site-packages\micro_sam\sam_annotator\annotator_2d.py", line 11, in <module>
    from .. import instance_segmentation, util
  File "D:\miniconda3_windows\envs\micro_sam\lib\site-packages\micro_sam\instance_segmentation.py", line 36, in <module>
    from .prompt_based_segmentation import segment_from_mask
  File "D:\miniconda3_windows\envs\micro_sam\lib\site-packages\micro_sam\prompt_based_segmentation.py", line 311, in <module>
    original_size: Optional[tuple[int, ...]] = None,
TypeError: 'type' object is not subscriptable
```

**Comment by constantinpape** [comment]
The problem is due to the fact that type annotations work differently in python 3.8, see
https://stackoverflow.com/questions/75719006/in-python-for-parameter-typing-do-you-use-the-capital-versions-of-dict-and-list

The PR in #213 should fix this, and once it's merged it should work in python 3.8, so you will only need to update the micro_sam environment then. 
If you then still run into issues with type annotations please report them here, or alternatively use an environment with python >= 3.9.

**Comment by phamtronglam** [comment]
Thanks a  lot. Updating to python 3.9 worked.
